lib/datadog/appsec/configuration/settings.rb

```diff
@@ -113,6 +114,7 @@ def duration(base = :ns, type = :integer)
           'DD_APPSEC_TRACE_RATE_LIMIT' => [:trace_rate_limit, Settings.integer],
           'DD_APPSEC_OBFUSCATION_PARAMETER_KEY_REGEXP' => [:obfuscator_key_regex, Settings.string],
           'DD_APPSEC_OBFUSCATION_PARAMETER_VALUE_REGEXP' => [:obfuscator_value_regex, Settings.string],
+          'DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING' => [:automated_track_user_events, Settings.string],
```

Comment:
Not too fond of that naming. Is `DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING` a specified env var?

Reply:
Yes is part of the RFC

Reply:
Yes is part of the RFC